frontend/packages/knative-plugin/console-extensions.json

```diff
@@ -1124,5 +1124,369 @@
     "flags": {
       "required": ["KNATIVE_SERVING_SERVICE", "KNATIVE_SERVING_REVISION", "KNATIVE_EVENTING"]
     }
+  },
```

Comment:
@vikram-raj can we remove model definition and static plugins now as we have dynamic one's now

https://github.com/openshift/console/blob/master/frontend/packages/knative-plugin/src/plugin.tsx#L38-L43
https://github.com/openshift/console/blob/master/frontend/packages/knative-plugin/src/models.ts#L1-L482